charts/datarepo/ci/testing-values.yaml

```diff
@@ -16,9 +16,6 @@ datarepo-api:
   fullnameOverride: test-datarepo-api
   image:
     tag: 1.0.180
-  shutdownTimeoutSeconds: 3
-  readinessInitialDelaySeconds: 15
-  livenessInitialDelaySeconds: 20
```

Comment:
Per @tcjiang9 , this will now default to the real deployment values, which are much longer.